<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QMainWindow" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>594</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	border-style: flat;
	background: transparent;
}</string>
    </property>
    <property name="text">
     <string>主题设置</string>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/line.png</normaloff>:/img/img/line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 默认 */
QPushButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 30px;                            /* 控件最小宽度 */
    min-height: 30px;
	background-color: rgb(255，251，240);
	border-radius:15px;
}

/* 鼠标悬停 */
QPushButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QPushButton:pressed,QToolButton:checked{
	border-left: 3px  rgb(93, 95, 97);
	background-color: rgb(245, 212, 217);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/day.png</normaloff>:/img/img/day.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_black">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 默认 */
QPushButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 30px;                            /* 控件最小宽度 */
    min-height: 30px;
	background-color: rgb(255，251，240);
	border-radius:15px;
}

/* 鼠标悬停 */
QPushButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QPushButton:pressed,QToolButton:checked{
	border-left: 3px  rgb(93, 95, 97);
	background-color: rgb(245, 212, 217);	
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/night.png</normaloff>:/img/img/night.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	border-style: flat;
	background: transparent;
}</string>
    </property>
    <property name="text">
     <string>输出设置</string>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/line.png</normaloff>:/img/img/line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>默认保存位置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>250</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 30px;                            /* 控件最小宽度 */
    min-height: 30px;
	border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>选择文件夹</string>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/files.png</normaloff>:/img/img/files.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	border-style: flat;
	background: transparent;
}</string>
    </property>
    <property name="text">
     <string>语言设置</string>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/line.png</normaloff>:/img/img/line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中文</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>英语</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	border-style: flat;
	background: transparent;
}</string>
    </property>
    <property name="text">
     <string>模型选择</string>
    </property>
    <property name="icon">
     <iconset resource="../resource.qrc">
      <normaloff>:/img/img/line.png</normaloff>:/img/img/line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
